misc/checkout-pr.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
+
+if [ $# -ne "1" ]; then
+	echo "usage: $0 <PR number>"
+	exit 1
+fi
+
+if [ ! -x "$(command -v curl)" ]; then
+	echo "You need curl to run this script."
+	exit 1
+fi
+
+pr=$1
+pr_json="pr.json"
+
+curl -o $pr_json https://api.github.com/repos/namhyung/uftrace/pulls/$pr
+repo=$(jq -r '.head.repo.html_url' $pr_json)
```

Comment:
What is `jq`?

Reply:
https://jqlang.github.io/jq

Reply:
You can easily understand by running the following command. It's to get the PR sender's uftrace repo.
```
$ curl -s https://api.github.com/repos/namhyung/uftrace/pulls/1947 | jq -r '.head.repo.html_url'
https://github.com/honggyukim/uftrace
```

Reply:
We can also get the branch name as follows.
```
$ curl -s https://api.github.com/repos/namhyung/uftrace/pulls/1947 | jq -r '.head.ref'
check/checkout-pr.sh
```

Reply:
That's cool.  Is it ok to assume if it's installed on all modern distros?

Reply:
Sorry for the late answer but I guess yes.